ViewModel/ImpactAreaScenario/Results/PerformanceLongTermRiskVM.cs

```diff
@@ -6,51 +6,30 @@ namespace HEC.FDA.ViewModel.ImpactAreaScenario.Results
 {
     public class PerformanceLongTermRiskVM : PerformanceVMBase
     {
-
-        public PerformanceLongTermRiskVM(List<ThresholdComboItem> metrics)
+        public PerformanceLongTermRiskVM(metrics.Results iasResult, List<ThresholdComboItem> metrics)
```

Comment:
Rename your combo item variable to something other than metrics, it's a namespace